pkg/identity/gitlabcom/principal_test.go

```diff
@@ -263,16 +336,14 @@ func TestEmbed(t *testing.T) {
 	}
 }
 
-func factIssuerIs(issuer string) func(x509.Certificate) error {
-	return factExtensionIs(asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 57264, 1, 1}, issuer)
-}
-
 func factExtensionIs(oid asn1.ObjectIdentifier, value string) func(x509.Certificate) error {
 	return func(cert x509.Certificate) error {
 		for _, ext := range cert.ExtraExtensions {
 			if ext.Id.Equal(oid) {
-				if !bytes.Equal(ext.Value, []byte(value)) {
```

Comment:
How come this needed to change?

Reply:
This function was previously unused in the tests. When I tried to use it, this comparison was failing. I can't speak to why, I just copied the working implementation from here: https://github.com/sigstore/fulcio/blob/main/pkg/identity/github/principal_test.go#L644

Reply:
ah gotcha! sgtm 👍

Reply:
The reason is a change in the format to now be asn1 encoded strings rather than just raw values.